README

```diff
@@ -33,7 +33,7 @@ packages installed:
 	apt install --no-install-recommends --no-install-suggests \
 		parted cgpt \
 		git gawk device-tree-compiler vboot-kernel-utils gcc-arm-none-eabi \
-			u-boot-tools
+		u-boot-tools m4 patch \
```

Comment:
As far as I understand these two are needed to build the ath9k_htc firmware - if so, move them alongside cmake